The_Glowing_Volcano/Assets/Scripts/InventoryObject.cs

```diff
@@ -45,14 +57,19 @@ public class InventoryObject : InteractiveObject
     private Light lighting;
     private int childCount;
     private Transform passengerTransform;
+    private bool hasBeenUsed = false;
 
     protected virtual void Start()
     {
         renderer = GetComponent<Renderer>();
         collider = GetComponent<Collider>();
         lighting = GetComponent<Light>();
         childCount = transform.childCount;
-        passengerTransform = objectToTransport.GetComponent<Transform>();
+
+        if (transportItem)
```

Comment:
@ThreeEyedVenelope  @ifaith1 bool variables should be named as yes or no questions.